testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/mgmt/JMSBridgeManagementTestCase.java

```diff
@@ -0,0 +1,360 @@
+    }
+
+    @Override
+    protected ModelControllerClient getModelControllerClient() {
+        return this.managementClient.getControllerClient();
+    }
+
+    public void reload() throws Exception {
+        executeReloadAndWaitForCompletion(this.managementClient.getControllerClient(), 100000);
+    }
+    
+    
+    static class SetupConnectionFactory implements ServerSetupTask {
+
+        @Override
+        public void setup(final ManagementClient managementClient, final String containerId) throws Exception {
+            final ModelNode operation = Util.createAddOperation(ADDRESS_XA_CONNECTION_FACTORY);
+            operation.get("factory-type").set("XA_GENERIC");
+            //operation.get("connector", "netty").set("netty");
+            operation.get(CommonAttributes.CONNECTORS).add("netty");
+            operation.get("entries").add("java:jboss/jms/XAConnectionFactory")
+                    .add("java:jboss/exported/jms/XAConnectionFactory");
+
+            ModelNode result = managementClient.getControllerClient().execute(operation);
+            Assert.assertTrue(result.toString(), Operations.isSuccessfulOutcome(result));
+        }
+
+        @Override
+        public void tearDown(final ManagementClient managementClient, final String containerId) throws Exception {
+            managementClient.getControllerClient().execute(Util.createRemoveOperation(ADDRESS_XA_CONNECTION_FACTORY));
+        }
+    }
+
+    class SetupQueuesFactory implements ServerSetupTask {
+
+        @Override
+        public void setup(final ManagementClient managementClient, final String containerId) throws Exception {
+            ModelNode result = setUpQueue(managementClient.getControllerClient(), "sourceQueue", ADDRESS_QUEUE_SOURCE);
+            Assert.assertTrue(result.toString(), Operations.isSuccessfulOutcome(result));
+            result = setUpQueue(managementClient.getControllerClient(), "targetQueue", ADDRESS_QUEUE_TARGET);
+            Assert.assertTrue(result.toString(), Operations.isSuccessfulOutcome(result));
+        }
+
+        @Override
+        public void tearDown(final ManagementClient managementClient, final String containerId) throws Exception {
+            managementClient.getControllerClient().execute(Util.createRemoveOperation(ADDRESS_QUEUE_SOURCE));
+            managementClient.getControllerClient().execute(Util.createRemoveOperation(ADDRESS_QUEUE_TARGET));
+        }
+
+        private ModelNode setUpQueue(final ModelControllerClient client, final String name, final PathAddress address)
+                throws IOException {
+            final ModelNode operation = Util.createAddOperation(address);
+            operation.get("entries").add("java:jboss/jms/queue/" + name).add("java:jboss/exported/jms/queue/" + name);
+            operation.get("durable").set(true);
+            return client.execute(operation);
+
+        }
+    }
+
+    class DisableHornetQSecruity implements ServerSetupTask {
```

Comment:
is it really necessary to disable security for this test? (instead of setting up the user credentials in the jms-bridge configuration)?